tutorial/input_output.py

```diff
@@ -0,0 +1,234 @@
+                "It concatenates paths.": "Correct, it lets you construct a path from different segments",
+            },
+            correct_answer="It concatenates paths",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="If you use Pathlib,  do you need to use different path separators on Windows and Linux to combine path segments?",
+            options={
+                "No, you can combine Pathlib.Path objects with <code>/</code>": "Correct! Pathlib will then generate the correct path for your OS.",
+                "Yes.": "Wrong! You can always use  <code>/</code>",
+            },
+            correct_answer="No, you can combine Pathlib.Path objects with <code>/</code>",
+            shuffle=True,
+        )
+
+        q3 = Question(
+            question="""The path </code>Pathlib.Path("./")</code> represent a relative path. What location does it refer to?""",
+            options={
+                "Relative to the current working directory, which is the location of the current Python script being run.": "Correct!",
+                "Relative to the user's home directory": "Wrong!",
+            },
+            correct_answer="Relative to the current working directory, which is the location of the current Python script being run.",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2, q3])
+
+
+class TextFiles(Quiz):
+    def __init__(self, title=""):
+        q1 = Question(
+            question="Can you read from a file before calling <code>open</code>?",
+            options={
+                "Yes": "Wrong, if the file is not open, we cannot access its contents.",
+                "No": "Correct, we need to open the file first.",
+            },
+            correct_answer="No",
+            shuffle=True,
+        )
+
+        q2 = Question(
+            question="The function <code>readlines</code> reads the entire content of a text file into <b>one</string>. Is this correct?",
+            options={
+                "No": "Correct! It reads the file line by line.",
+                "Yes": "Wrong! It reads the file line by line and returns a list of <code>str</code>, with one element for each line.",
+            },
+            correct_answer="No",
+            shuffle=True,
+        )
+
+        q3 = Question(
+            question="""The path </code>Pathlib.Path("./")</code> represent a relative path. What location does it refer to?""",
+            options={
+                "Relative to the current working directory, which is the location of the current Python script being run.": "Correct!",
+                "Relative to the user's home directory": "Wrong!",
+            },
+            correct_answer="Relative to the current working directory, which is the location of the current Python script being run.",
+            shuffle=True,
+        )
+
+        super().__init__(questions=[q1, q2, q3])
+
+
+class ReadFiles(Quiz):
```

Comment:
The questions from this quiz are the same as in the previous one. Is it intended?